src/AdminMenus/FormGlobalSettingsAdminSubMenu.php

```diff
@@ -173,13 +173,18 @@ protected function processAttributes($attr): array
 	{
 		$type = isset($_GET['type']) ? \sanitize_text_field(\wp_unslash($_GET['type'])) : SettingsGeneral::SETTINGS_TYPE_KEY; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 
+		$settingsSidebarOutput = [];
+		foreach ($this->settingsGlobal->getSettingsSidebar($type) as $item) {
+			$settingsSidebarOutput[$item['type'] ?? 'general'][] = $item;
```

Comment:
```suggestion
			$sidebarType = $item['type'] ?? 'general';
			$settingsSidebarOutput[$sidebarType][] = $item;
```

I find this to be a bit more readable. At the first look I thought that was some syntax error 😂